bundles/org.openhab.binding.satel/README.md

```diff
@@ -55,6 +55,8 @@ Example:
 Bridge satel:ethm-1:home [ host="192.168.0.2", refresh=1000, userCode="1234", encryptionKey="abcdefgh" ]
 ```
 
+**NOTE:** There can be only one client connected to ETHM-1 module. It does not accept new connections if there is already a connection established. In case you have troubles connecting to the system using this module, please make sure there is no other client (for example installed 1.x version of the binding) already connected to it.
```

Comment:
Please use a new line for each sentence. They are  concatenated automatically.

Reply:
Done.